src/main/java/com/google/sps/servlets/AddOrganizationServlet.java

```diff
@@ -52,15 +60,31 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     openHours.add(Integer.parseInt(hourOpen));
     openHours.add(Integer.parseInt(hourClosed));
 
-    ArrayList<String> moderatorList = new ArrayList<String>();
-    moderatorList.add("anon creator"); //dummy value for before auth gets set up
+    /* This implementation stores history entries as embedded entities instead of custom objects
+     * because it is much simpler that way */
+    ArrayList changeHistory = new ArrayList<>();
+    UserService userService = UserServiceFactory.getUserService();
+    boolean isUserLoggedIn = userService.isUserLoggedIn();
+    String username;
+    if (isUserLoggedIn) {
+      /* Currently uses user email to be consistent w other parts of codebase, subject to change */
+      username = userService.getCurrentUser().getEmail();
```

Comment:
Is there a issue tracking this? Stylistically, it'd be better to add a TODO(<issue number>): description so that this type of debt can be tracked and fixed.

Reply:
I changed it to email because the current other PR's are referencing users by their email, so I designed this system to reference them in the same way

Reply:
Right, but should other parts of the codebase be doing this? What's the plan to fix it, and how does this new code fit into that plan?

Reply:
Can we talk about userId vs email as a pod later today? Jenny Sarah and I were thinking that for an MVP it would make sense to just reference users by full email address since that can't be the same for multiple different users, but I know there are some drawbacks related to that. I think for this feature though I'm going to change it to record the userId and the email.